### DISCUSSION TITLE
executing the CheckpointSave node

### DISCUSSION THREAD

**Discussion by joshgura** [Comfy-Org/ComfyUI#4109] [Q&A]
I have a unet, text_encoder clips and a vae. I hooked them all up with CheckpointSave and hit Queue, but my checkpoint isn't being output to the /checkpoints folder. What's the procedure to cause the node CheckpointSave to save the checkpoint?

edit: i just read somewhere that there needs to be a default workflow with a simple prompt in the project in order for 'queue' to trigger the function and now something is happening.

**Comment by comfyanonymous** [comment]
It gets saved to the output/checkpoints folder.
